game/models.py

```
@@ -363,3 +368,40 @@ def save(self, *args, **kwargs):
+class GameRecord(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True, blank=True)
+    session_key = models.CharField(max_length=40, db_index=True)
+    white_label = models.CharField(max_length=64, default="White")
+    black_label = models.CharField(max_length=64, default="Black")
+    result = models.CharField(max_length=7, default="*")
+    termination = models.CharField(max_length=32, default="unknown")
+    pgn = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    expires_at = models.DateTimeField(default=_expires_at_default, db_index=True)
+    
+    def save(self, *args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing type annotation for `*args`

(ANN002)